src/codegen/metadata/functions.py

```diff
@@ -12837,7 +12837,7 @@
             },
             {
                 'ctypes_data_type': 'ctypes.c_uint32',
-                'direction': 'in',
+                'direction': 'out',
```

Comment:
I think this will remove the `arraySize` field from `GetAnalogPowerUpStatesWithOutputTypeRequest`. Instead, set `pointer=True` for this parameter.